Dockerfile

```diff
@@ -18,6 +18,7 @@ RUN bundle install
 
 COPY requirements.txt .
 RUN python3 -m venv /venv && . /venv/bin/activate && \
+    python3 -m pip install wheel==0.42.0 && \
```

Comment:
For my learning - is it good practice to pin the wheel version when installing requirements in a containerized environment?